packages/driver-kube-pod/src/driver/client/index.ts

```diff
@@ -57,15 +57,19 @@ const ensureSingleDockerHostDeployment = (): ApplyFilter => {
   }
 }
 
-const kubeClient = ({ log, namespace, kc, profileId, template, package: packageDetails, kubeconfig }: {
+const kubeClient = ({ log, namespace, kc, profileId, template, package: packageDetails, kubeconfig, context }: {
   log: Logger
   kc: k8s.KubeConfig
   kubeconfig?: string
+  context?: string
   namespace: string
   profileId: string
   template: Buffer | string | Promise<Buffer | string>
   package: Package | Promise<Package>
 }) => {
+  if (context) {
```

Comment:
unexpected side effect, better to set in caller

Reply:
done